packages/shared/src/components/notifications/NotificationIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import {
+  notificationDefaultTheme,
+  notificationIcon,
+  NotificationIcon,
+} from '../../graphql/notifications';
+
+const noBackgroundIcons = [
+  NotificationIcon.DailyDev,
+  NotificationIcon.CommunityPicks,
+];
+
+interface NotificationItemIconProps {
+  icon: NotificationIcon;
+}
+
+function NotificationItemIcon({
+  icon,
+}: NotificationItemIconProps): ReactElement {
+  const Icon = notificationIcon[icon] ?? notificationIcon.DailyDev;
+
+  if (!notificationIcon[icon] || noBackgroundIcons.indexOf(icon) !== -1) {
```

Comment:
`noBackgroundIcons.includes(icon)` is easier to read

Reply:
Good point. Let's do this. I kinda forgot this API all the time 😆